main.md

```diff
@@ -253,23 +253,24 @@ subset of the same mapping).
 
 An SD-JWT is a JWT that MUST be signed using the issuer's private key. The
 payload of an SD-JWT MUST contain the `sd_digests` and `hash_alg` claims
-described in the following, and MAY contain a holder's public key or a reference
+described in the following sections, and MAY contain a holder's public key or a reference
```

Comment:
I chose "in the following" for being less ambiguous, as "following section" can be read as "go to the next top-level section to read more", but we want to say "in the following subsections", which is an awkward way to word it. Therefore just "in the following". But since this is the second time someone suggests this edit, I'm starting to question my interpretation here....

Reply:
let's just say `in the following subsections` or `in this document` than!